api/src/main/java/net/kyori/adventure/text/format/NamedTextColor.java

```diff
@@ -156,7 +156,10 @@ public final class NamedTextColor implements TextColor {
    */
   public static final NamedTextColor WHITE = new NamedTextColor("white", WHITE_VALUE);
 
-  private static final List<NamedTextColor> VALUES = Collections.unmodifiableList(Arrays.asList(BLACK, DARK_BLUE, DARK_GREEN, DARK_AQUA, DARK_RED, DARK_PURPLE, GOLD, GRAY, DARK_GRAY, BLUE, GREEN, AQUA, RED, LIGHT_PURPLE, YELLOW, WHITE));
+  /**
+   * @since 4.14.0
+   */
+  public static final List<NamedTextColor> VALUES = Collections.unmodifiableList(Arrays.asList(BLACK, DARK_BLUE, DARK_GREEN, DARK_AQUA, DARK_RED, DARK_PURPLE, GOLD, GRAY, DARK_GRAY, BLUE, GREEN, AQUA, RED, LIGHT_PURPLE, YELLOW, WHITE));
```

Comment:
Doesn't need to be made public - can be obtained from `NAMES.values()`

Reply:
`NAMES.values()` returns a `Set`.